cardano-cli/src/Cardano/CLI/Shelley/Commands.hs

```diff
@@ -118,7 +118,7 @@ data StakeAddressCmd
   = StakeAddressKeyGen VerificationKeyFile SigningKeyFile
   | StakeAddressKeyHash (VerificationKeyOrFile StakeKey) (Maybe OutputFile)
   | StakeAddressBuild StakeVerifier NetworkId (Maybe OutputFile)
-  | StakeRegistrationCert StakeVerifier OutputFile
+  | StakeRegistrationCert (Either StakeVerifier StakeAddress) OutputFile
```

Comment:
A more seamless change would be to add an additional constructor to the `StakeVerifier` data type that allows  the specification of a `StakeAddress`.

Reply:
done